src/components/core/tests/GeoChart.spec.tsx

```diff
@@ -68,14 +68,52 @@ describe("GeoChart", () => {
 
     it("should call onDataTooLarge", () => {
         const onDataTooLarge = jest.fn();
-        renderComponent({
-            config: {
-                limit: 20,
-                mapboxToken: "",
-                mdObject,
-            },
+        const initProps: Partial<IGeoChartInnerProps> = {
+            config: { ...getGeoConfig({ isWithLocation: true }), limit: 5, mapboxToken: "" },
+            execution: null,
+            onDataTooLarge,
+        };
+        const props: Partial<IGeoChartInnerProps> = {
+            config: { ...getGeoConfig({ isWithLocation: true }), limit: 5, mapboxToken: "" },
             onDataTooLarge,
-        });
-        expect(onDataTooLarge).toHaveBeenCalledTimes(1);
+            execution: {
+                executionResponse: getExecutionResponse(true),
+                executionResult: getExecutionResult(true),
+            },
+        };
+        const component = renderComponent(initProps);
+        component.setProps(props);
+        expect(onDataTooLarge).toBeCalled();
+    });
+
+    it("should render GeoChartInner with expected state", () => {
```

Comment:
"expected state" seems general to me.
Please give more details.

Reply:
This case will assert state of GeoChartInner (enabledLegendItems) after component updated

Reply:
I mean test name should be more details.

Reply:
Or please check there is similar cases in `StaticLegend`.
If it is, we can skip this.

Reply:
fixed